homeassistant/components/switch/tellstick.py

```diff
@@ -31,16 +30,23 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             "Failed to import tellcore")
         return
 
+    # pylint: disable=no-member
+    if telldus.TelldusCore.callback_dispatcher is None:
```

Comment:
The dispatcher was set globally in the library because it can only be one dispatcher connected to the lib per program. The component that creates the first TelldusCore instance sets it and it will be reused in the other component!